notebooks/OFC/truncation_thresholds.ipynb

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
<p>A short explanation of what's happening in each cell would be nice too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>